mod/reportedcontent/start.php

```diff
@@ -19,26 +19,10 @@ function reportedcontent_init() {
 	elgg_extend_view('elgg.css', 'reportedcontent/css');
 	elgg_extend_view('admin.css', 'reportedcontent/admin_css');
 
-
-	if (elgg_is_logged_in()) {
-		elgg_require_js('elgg/reportedcontent');
-
-		// Extend footer with report content link
-		elgg_register_menu_item('extras', array(
-			'name' => 'report_this',
-			'href' => 'reportedcontent/add',
-			'title' => elgg_echo('reportedcontent:this:tooltip'),
-			'text' => elgg_view_icon('report-this'),
-			'priority' => 500,
-			'section' => 'default',
-			'link_class' => 'elgg-lightbox',
-		));
-	}
-
+	elgg_register_plugin_hook_handler('register', 'menu:extras', 'reportedcontent_extras_menu');
```

Comment:
I think it is safer to make a PR against master at this time, and call it out in docs.